### ISSUE TITLE
OmniXAI Showing "StagingError" error when using Shap Technique with Batchnormlayer

### ISSUE THREAD

**Issue by Fardeenkhan1** [salesforce/OmniXAI#97]
`model = tf.keras.models.Sequential()
model.add(tf.keras.layers.Conv2D(32, kernel_size=(3, 3), activation='relu', input_shape=input_shape))
model.add(tf.keras.layers.Conv2D(64, (3, 3), activation='relu'))
model.add(tf.keras.layers.MaxPooling2D(pool_size=(2, 2)))

model.add(tf.keras.layers.BatchNormalization()) 

> after removing this batchnorm layer it is working

model.add(tf.keras.layers.Dropout(0.1))
model.add(tf.keras.layers.Flatten())
model.add(tf.keras.layers.Dense(128, activation='relu'))
model.add(tf.keras.layers.Dropout(0.1))
model.add(tf.keras.layers.Dense(num_classes))`
i am trying to test this model with Shap but it is giving "Stagingerror" if i remove the BatchNormalization layer then it is working.

return this error
![image](https://github.com/salesforce/OmniXAI/assets/60724602/c7816435-ad71-41d2-8967-de09204be94c)
